flixel/system/frontEnds/BitmapFrontEnd.hx

```diff
@@ -114,7 +114,7 @@ class BitmapFrontEnd
 	/**
 	 * Check the local bitmap cache to see if a bitmap with this key has been loaded already.
 	 *
-	 * @param	Key		The string key identifying the bitmap.
+	 * @param	Key	The key identifying the bitmap.
```

Comment:
Doc params should:
- start with lowercase letters
- have 3 spaces before the arg name and at least 2 after
- have all arg descriptions start on the same column
Other function doc rules:
- `@return` should have 2 spaces after, so the description aligns with the params
- Types should be wrapped in `backticks`
- Avoid types in args with default values (`unique:Bool = false` can be `unique = false`)